curriculum/challenges/english/25-front-end-development/quiz-responsive-web-design/66ed9034f45ce3ece4053ebd.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 1
+`em`
 
 ---
 
-Placeholder distractor 2
+`px`
 
 ---
 
-Placeholder distractor 3
+`rem`
 
 #### --answer--
 
-Placeholder answer
+`vw`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does mobile-first design emphasize?
 
 #### --distractors--
 
-Placeholder distractor 1
+Designing for desktops and scaling down for smaller devices.
 
 ---
 
-Placeholder distractor 2
+Using only desktop breakpoints.
 
 ---
 
-Placeholder distractor 3
+Focusing on the highest resolution devices first.
 
 #### --answer--
 
-Placeholder answer
+Designing for smaller screens first and scaling up for larger devices.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which media query will apply styles when the device's width is between 600px and 1200px?
 
 #### --distractors--
 
-Placeholder distractor 1
+`@media screen and (max-width: 1200px)`
 
 ---
 
-Placeholder distractor 2
+`@media screen and (min-width: 600px)`
 
 ---
 
-Placeholder distractor 3
+`@media screen and (width: 800px)`
 
 #### --answer--
 
-Placeholder answer
+`@media screen and (min-width: 600px) and (max-width: 1200px)`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the default flex-direction value in Flexbox?
 
 #### --distractors--
 
-Placeholder distractor 1
+`column`
 
 ---
 
-Placeholder distractor 2
+`row-reverse`
 
 ---
 
-Placeholder distractor 3
+`column-reverse`
 
 #### --answer--
 
-Placeholder answer
+`row`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In Flexbox, how do you distribute space between items in a container?
 
 #### --distractors--
 
-Placeholder distractor 1
+`align-items`
 
 ---
 
-Placeholder distractor 2
+`flex-grow`
 
 ---
 
-Placeholder distractor 3
+`flex-wrap`
 
 #### --answer--
 
-Placeholder answer
+`justify-content`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you apply a grid layout to a container in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+`display: flexbox;`
 
 ---
 
-Placeholder distractor 2
+`display: block;`
 
 ---
 
-Placeholder distractor 3
+`display: table;`
 
 #### --answer--
 
-Placeholder answer
+`display: grid;`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `fr` unit represent in CSS Grid?
 
 #### --distractors--
 
-Placeholder distractor 1
+Flexible ratio
 
 ---
 
-Placeholder distractor 2
+Frame ratio
 
 ---
 
-Placeholder distractor 3
+Fixed region
 
 #### --answer--
 
-Placeholder answer
+Fraction of available space
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a valid media query for targeting devices with a resolution of 2x or higher?
 
 #### --distractors--
 
-Placeholder distractor 1
+`@media (resolution: 2x)`
 
 ---
 
-Placeholder distractor 2
+`@media (max-resolution: 2x)`
 
 ---
 
-Placeholder distractor 3
+`@media screen and (min-width: 1200px)`
 
 #### --answer--
 
-Placeholder answer
+`@media (min-resolution: 2dppx)`
```

Comment:
Shouldn't this be `px`? `dppx` don't seem to be able to ensure what the actual resolution will be.

```suggestion
`@media (min-resolution: 2px)`
```